api/envoy/config/wasm/v2alpha/wasm.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+package envoy.config.wasm.v2alpha;
+
+import "envoy/api/v2/core/base.proto";
+
+import "google/protobuf/any.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.wasm.v2alpha";
+option java_outer_classname = "WasmProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package = "envoy.extensions.wasm.v3alpha";
+
+// [#protodoc-title: Wasm service]
+
+// Configuration for a Wasm VM.
+// [#next-free-field: 6]
+// [#not-implemented-hide:] pending implementation.
+message VmConfig {
+  // An ID which will be used along with a hash of the wasm code (or null_vm_id) to determine which
```

Comment:
Nit: Wasm code. Also, for constants, please write `*null_vm_id` (is this a string? it should be `*"null_vm_id"*`)

Reply:
null_vm_id is not a constant.  It is something that the user provides with a Null VM plugin which uniquely defines it.  It is the name it uses to register the null vm plugin.  Updated comments.